R/write.R

```diff
@@ -8,6 +8,8 @@
 #' @param path Path where transport file will be written. File name sans will be
 #'   used as `xpt` name.
 #' @param label Dataset label. It must be<=40 characters.
+#' @param no_fail If TRUE, xpt validation will report a warning instead of an
+#'   error on dataset validation.
```

Comment:
Should we say anything to user about the default being FALSE?

Reply:
If you do keep the argument, I would consider changing the name. Reading `no_fail=FALSE` is a double negative and just a bit more difficult to comprehend for users. Also, I think most users would prefer to see warnings compared to error, so I would make warning messages the default value. 🍁

Reply:
Updated the argument to 'strict_checks' and flipped the default